auto_inertial_params/proposal.md

```diff
@@ -0,0 +1,149 @@
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>
+
+# Automatically Compute Inertia Tensor Proposal
+
+* **Authors**:
+Jasmeet Singh `<jasmeet0915@gmail.com>`, Addisu Taddese `<addisu@openrobotics.org>`, Dharini Dutia `<dharinidutia@openrobotics.org>`
+* **Status**: Draft
+* **SDFormat Version**: 1.11
+* **`libSDFormat` Version**: 14.X 
+
+## Introduction
+
+This proposal suggests adding new attributes and elements to support the automatic calculation of [Moments of Inertia and Products of Inertia](https://en.wikipedia.org/wiki/List_of_moments_of_inertia) for a link in SDFormat 1.11. It also proposes adding support for parsing these elements and attributes in libsdformat14.
+
+Setting physically plausible values for inertial parameters is crucial for an accurate simulation. However, these parameters are often complex to comprehend and visualize. And users may tend to enter wrong values leading to incorrect simulation. Therefore, native support for calculating inertial parameters through SDF specification would enable accurate simulations in simulators that use SDFormat.
```

Comment:
nit: SDF -> SDFormat.

Reply:
Corrected in d240e64041c050fe6b5af3c3e1310c907c106625